src/command/dev/router/run.rs

```diff
@@ -322,8 +325,18 @@ impl RunRouter<state::Watch> {
 
         let composition_messages =
             tokio_stream::StreamExt::filter_map(composition_messages, |event| match event {
+                CompositionEvent::Error(CompositionError::Build { source, .. }) => {
+                    let number_of_subgraphs = source.len();
+                    let error_to_output = RoverError::from(RoverClientError::BuildErrors {
+                        source,
+                        num_subgraphs: number_of_subgraphs,
+                    });
+                    eprintln!("{}", error_to_output);
+                    None
+                }
                 CompositionEvent::Error(err) => {
                     tracing::error!("Composition error {:?}", err);
+                    errln!("Error occurred when composing supergraph\n{}", err);
```

Comment:
Nit: You have two different print formats here.

Reply:
That's deliberate, as it apes the format of the original printing. Which is a pain but oh well